pandas/__init__.py

```diff
@@ -190,36 +191,17 @@
     __git_version__ = v.get("full-revisionid")
     del get_versions, v
 
-# DeprecationWarning for missing pyarrow
-from pandas.compat.pyarrow import pa_version_under10p1, pa_not_found
-
-if pa_version_under10p1:
-    # pyarrow is either too old or nonexistent, warn
-    from pandas.compat._optional import VERSIONS
-
-    if pa_not_found:
-        pa_msg = "was not found to be installed on your system."
-    else:
-        pa_msg = (
-            f"was too old on your system - pyarrow {VERSIONS['pyarrow']} "
-            "is the current minimum supported version as of this release."
-        )
-
+# GH#55043 - deprecation of the data_manager option
+if "PANDAS_DATA_MANAGER" in os.environ:
```

Comment:
If targeting main, this should probably remain removed

Reply:
thx. done